src/main/resources/application.properties

```diff
@@ -1,12 +1,4 @@
 server.port=8080
+spring.profiles.active=dev
```

Comment:
It's not recommended to set the active profile for everyone.

Reply:
corrected